net/ntpd/Makefile

```diff
@@ -107,6 +109,7 @@ CONFIGURE_ARGS += \
 	--enable-linuxcaps \
 	--with-yielding-select=yes \
 	--with-crypto \
+	$(if $(CONFIG_PACKAGE_ntp-utils),--with-sntp,--without-sntp) \
```

Comment:
this should be unconditional.

Reply:
Why?  If it's always enabled, it pulls in the pthread dep.  I'm more curious why this PR is still needed, wasn't this fixed alreayd?

Reply:
@neheb only the sntp binary requires the libevent2-pthreads dependency, so if ntp-utils is not selected, the binary will not be built and the libevent2-pthreads library for it won't be pulled. --with-sntp is selected by default during configure, so the with/without is necessary.

My initial #10308 PR added the libevent2-pthreads dep for the whole ntpd package, this PR makes the dependency selective only for the binary that actually requires it. I consider it an improvement as I think minimizing the amount of (unnecessary) libraries on an embedded is a important goal.